src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -167,6 +167,12 @@ public void canCheckOutUsingItemAndUserBarcode()
     assertThat("location is taken from holding",
       loan.getJsonObject("item").getJsonObject("location").getString("name"),
       is("3rd Floor"));
+
+    IndividualResource savedLoan = loansStorageClient.get(response);
```

Comment:
Should the response from the check out contain the name / description of the patron group at check out?

Reply:
@marcjohnson-kint This is a really good question. I don`t know the best way, but I think that in checkout response should be "patronGroupIdAtCheckout" because this one will be stored in mod-circulation-storage. 

What do you think about it?

Reply:
To me, the user performs a check out, and the patron group information should be part of that response, in a similar manner to when a loan is retrieved later (either individually or as part of a collection).

In the current version of this pull request, this is the `patronGroupAtCheckout` property. My understanding from previous comments is that is because the name is what is needed.

Does my reasoning make sense?

(I've provided my feedback on that design elsewhere in this pull request)

Reply:
@marcjohnson-kint Yes, I saw your comment about this.
So, we need to return "patronGroupAtCheckout" object with 'id' and 'name' fields.

Moreover, we need to return this one ("patronGroupAtCheckout" object) in the checkout response.

Am I right?

Reply:
> So, we need to return "patronGroupAtCheckout" object (with 'id' and 'name' fields) instead of the field "patronGroupAtCheckout".

My suggestion is to change it to an object. If the only property that is really needed is the `name` then it is ok to start with just that. Including `id` as well, might make testing easier.

> Moreover, we need to return this one ("patronGroupAtCheckout" object) in the checkout response.

My understanding is that the check out response includes the loan, so this should be included as part of the loan.

Reply:
@marcjohnson-kint I changed it in this way. Could you please review this it?

Reply:
This test does not appear to have a check for the patron group in the check out response. 

I thought that is what you were adding, have I missed something?

Reply:
You are right. 

I didn`t catch it for the first time.  Now, I`m adding it. 
Thank you!